api/package.json

```diff
@@ -2,10 +2,9 @@
   "name": "api",
   "version": "0.0.0",
   "scripts": {
-    "generate": "bunx prisma generate",
     "start": "bun --hot --inspect=ws://0.0.0.0:6499/forcingPrefix src/server.ts",
     "start:prod": "bun src/server.ts",
-    "postinstall": "bun generate"
+    "postinstall": "bunx prisma generate"
```

Comment:
Changing this because the script isn't resolved properly when adding packages in `api`